File: java/UserService.java
import java.util.List;

public class UserService {

    public static double calculateAverage(List<Integer> numbers) {
        int total = 0;

        for (int number : numbers) {
            total += number;
        }

        return total / numbers.size();
    }

    public static String findUserName(List<User> users, int id) {
        for (User user : users) {
            if (user.getId() == id) {
                return user.getName();
            }
        }

        return null;
    }

    public static boolean isAdult(int age) {
        if (age > 18) {
            return true;
        }

        return false;
    }

    public static void main(String[] args) {
        List<Integer> numbers = List.of();
        System.out.println(calculateAverage(numbers));
    }
}

class User {
    private int id;
    private String name;

    public User(int id, String name) {
        this.id = id;
        this.name = name;
    }

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }
}

File: python/user_service.py
import json

def calculate_average(numbers):
    total = 0

    for number in numbers:
        total += number

    return total / len(numbers)

def get_user_name(user_json):
    user = json.loads(user_json)
    return user["name"]

def find_user(users, user_id):
    for user in users:
        if user["id"] == user_id:
            return user

    return None

def process_scores(scores):
    average = calculate_average(scores)

    if average > 90:
        grade = "A"
    elif average > 80:
        grade = "B"
    elif average > 70:
        grade = "C"
    else:
        grade = "F"

    return grade

if __name__ == "__main__":
    scores = []
    print(calculate_average(scores))
